CHANGELOG.rst

```diff
@@ -11,6 +11,10 @@ in development
 Added
 ~~~~~
 
+* Use string lock name instead of object id. #5484
+
+  Contributed by @khushboobhatia01
+
```

Comment:
The changelog would need to go into `development` section as `v3.6.0` was already cut.
The record best fit for the `Bugfix` section. Please also mention that's it's a fix for a regression, referencing the #5358.
This way we'll keep the issues connected.